test/ng/locationSpec.js

```diff
@@ -57,13 +60,14 @@ describe('$location', function() {
         search: '',
         setAttribute: angular.noop
       };
-    }));
+    });
 
-    afterEach(inject(function($sniffer) {
-      if (msie) return;
+    afterEach(function() {
+      // Support: non-Windows browsers
+      if (msie || /\bEdge\/[\d\.]+\b/.test(window.navigator.userAgent)) return;
```

Comment:
No, it doesn't hurt. It is just about the mental overhead of "deciphering" `/\bEdge\/[\d\.]+\b/.test(window.navigator.userAgent)` vs `isEdge`.

> I'm a little afraid of making it too easy to use UA-sniffing, though.

Fair enough :smiley: